README.md

```diff
@@ -1,81 +1,74 @@
+    <br/>For example, on Linux, Mac, and Windows, open the [`wavefront.conf`](https://docs.wavefront.com/proxies_configuring.html#proxy-file-paths) file, add the line `otlpGrpcListenerPorts=4317`, and save the file.
+1. Configure your application to send trace data to the Wavefront Proxy. 
+    <br/>By default, OpenTelemetry SDKs send data over gRPC to `http://localhost:4317`.
+1. Explore the trace data using our [tracing dashboards](https://docs.wavefront.com/tracing_basics.html#visualize-distributed-tracing-data).
+
+### Send Data Using the OpenTelemetry Collector and the Wavefront Proxy
+If you have already configured your application to send data to the OpenTelemetry Collector, the data flows from your application to Tanzu Observability as shown in the diagram:
+
+**Note**: You need to use OpenTelemetry Collector Contrib version v0.28.0 or later to export traces to Tanzu Observability." 
+![Shows how the data flows from your application to the OpenTelemetry Collector to Tanzu Observability](images/opentelemetry_collector_tracing.png)
+
+Follow these steps:
+
+1. [Install the Wavefront Proxy](https://docs.wavefront.com/proxies_installing.html).
+      <ul>
+      <li>
+        Open port 30001, with <code>customTracingListenerPorts=30001</code>, for the proxy to generate span-level RED metrics.
+        </li>
+        <li>
+          Ensure that port 2878 is open to send spans and metrics to the Wavefront service. For example, on Linux, Mac, and Windows, open the <a href="proxies_configuring.html#proxy-file-paths"><code>wavefront.conf</code></a> file and confirm that <code>pushListenerPorts</code> is set to 2878, and that this configuration is uncommented. 
+        </li>
+      </ul>
+         
+1. Configure your application to send trace data to the OpenTelemetry Collector. See the [OpenTelemetry documentation](https://opentelemetry.io/docs/collector/) for details.
+1. Export the data from the OpenTelemetry Collector to the Tanzu Observability (Wavefront) trace exporter:
+    1. Create a directory to store all the files.
+    1. Download the binary from the latest release of the [OpenTelemetry Collector project](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases) to the directory you created.
+    1. In the same directory, create a file named `otel_collector_config.yaml`.
+    1. Copy the configurations in the [preconfigured YAML file](https://github.com/wavefrontHQ/opentelemetry-examples/blob/78f43e78b292c99bf00e6294712caf4ee940fc67/doc-resources/otel_collector_config.yaml) to the file you just created. For details on OpenTelemetry configurations, see [OpenTelemetry Collector Configuration](https://opentelemetry.io/docs/collector/configuration/).
+    1. On your console, navigate to the directory you created in the step above and run the following command to start OpenTelemetry Collector:
+        ```
+        ./otelcontribcol_darwin_amd64 --config otel_collector_config.yaml
```

Comment:
Do we want to change this to `./otelcol-contrib --config otel_collector_config.yaml`?

Reply:
Thank you! Good catch.